### PR TITLE
Investment Pie Graph

### PR DESCRIPTION
**Pull request by cindyzhang12**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?
- added pie graph to My Investments page which updates automatically with Airtable
- created new Table in Airtable for this

### How to Review
- check Investment.js, and InvestmentsPieGraph.js for the actual component
- you can also take a look at the new things in the schema and request.json() to see the new things implemented from adding a new table in Airtable!

### Next steps
- styling, need to figure out how to separate sections and add pointers like in Iris' designs

### Design Status

- lofi probably 

CC: @dfangshuo

### CODE REVIEW

File: src/styles/Investments.css [hunk 1]
```diff
@@ -279,3 +280,34 @@
 .investments-displayed-list {
   overflow: auto;
 }
+
+.investment-pie-graph {
+  width: 500px;
+}
+
+.Victory-Container {
```
Comment (dfangshuo) [inline src/styles/Investments.css:288]: hmm maybe rename this to investment-victory-chart-container or something?

File: src/styles/Investments.css [hunk 2]
```diff
@@ -279,3 +280,34 @@
...
+.financial-breakdown-graph-caption {
+  display: flex;
+  align-content: space-between;
+}
+
+.pie-graph-label-text {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+}
```
Comment (dfangshuo) [inline src/styles/Investments.css:313]: make these more specific, maybe prefix it with investment- or smth

File: src/screens/general/components/InvestmentsPieGraph.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+      return list;
+    });
+    const colorList = investmentBreakdowns.map(data => {
+      return data.color;
+    });
+    const labelStyle = {
+      fill: '#24364D',
+      fontSize: '20px'
+    };
```
Comment (dfangshuo) [inline src/screens/general/components/InvestmentsPieGraph.js:22]: move this to the top of the file

File: src/styles/Investments.css (hunk 1 above)
Comment (cindyzhang12) [reply src/styles/Investments.css:288]: this is like a default name they use from VictoryPie, I don't think I can change it!

File: src/screens/general/components/InvestmentsPieGraph.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { VictoryPie, VictoryLabel } from 'victory';
+
+export default class InvestmentsPieGraph extends React.PureComponent {
+  render() {
+    const { investmentBreakdowns } = this.props;
+    const labelStyle = {
+      fill: '#24364D',
```
Comment (dfangshuo) [inline src/screens/general/components/InvestmentsPieGraph.js:8]: declare this as a constant at the top of the file as well

**Pull request merged**